src/data/gallery.js

```diff
@@ -0,0 +1,716 @@
+export const gallerySections = [
+	{
+		"pk": 1,
+		"title": "Research Results",
+		"titleSlug": "research-results",
+		"description": "",
+		"order": 1
+	},
+	{
+		"pk": 2,
+		"title": "Fun Stuff",
+		"titleSlug": "fun-stuff",
+		"description": "",
+		"order": 4
+	},
+	{
+		"pk": 3,
+		"title": "Applications",
+		"titleSlug": "applications",
+		"description": "Applications using CAP data created by HLS staff, students, and affiliates.",
+		"order": 5
+	},
+	{
+		"pk": 4,
+		"title": "Coursework",
+		"titleSlug": "coursework",
+		"description": "",
+		"order": 2
+	},
+	{
+		"pk": 5,
+		"title": "Tutorials",
+		"titleSlug": "tutorials",
+		"description": "",
+		"order": 7
+	},
+	{
+		"pk": 6,
+		"title": "Applications (Third Party)",
+		"titleSlug": "applications-third-party",
+		"description": "Applications using CAP data created by third parties.",
+		"order": 6
+	},
+	{
+		"pk": 7,
+		"title": "Blog Posts",
+		"titleSlug": "blog-posts",
+		"description": "Third party explorations of CAP data.",
+		"order": 3
+	}
+]
+
+
+export const galleryItems = [
+	// {
+	//
+	// These internal items will likely not be reproduced.
+	// Shall we replace with a Perma Link? Some more complete web archive?
+	//
+	// 	"pk": 1,
+	// 	"title": "Wordclouds",
+	// 	"description": "Graphics showcasing the most-used words in California caselaw each year between 1852 and 2015.",
+	// 	"section": 2,
+	// 	"order": 5,
+	// 	"pageUrl": "https://case.law/exhibits/wordclouds",
+	// 	"repoUrl": "https://github.com/harvard-lil/cap-examples-old/blob/master/make_wordclouds.py"
+	// },
```

Comment:
I noticed that none of these define a value for an accompanying image — do you know where the images for sections like this is defined in the Django data? Is it easy to pull in? 

![Screenshot 2024-01-23 at 11 13 26 AM](https://github.com/harvard-lil/capstone-static/assets/4039311/4a00ee3e-2a19-44ce-8ac7-d56a1dcee9f5)

I don't think we need to address that in this PR, however — because we have the fallback image, that's so heavily used for so many other gallery items

Reply:
It also just occurred to me that many of these examples might be specific to things you can no longer do with the data — because they rely on the API. So, I'd hate to ask someone to put a bunch of effort into links that aren't actually going to be that useful!

Reply:
Ah, whoops! That's another thing that I meant to mention should get added to the list of Linear issues this AM.

_Yes_, I know where the images are... and to be discussed, how we want to get them in here. (I wanted to ask you about strategy, like, whether it makes any sense to base64 encode them and include that in the data file.... or whether that's not a good idea, and it would be better to dump them in an images directory.)

I'll move this convo to Linear!